config/crd/v1/bases/velero.io_rotationschemes.yaml

```diff
@@ -0,0 +1,128 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: rotationschemes.velero.io
+spec:
+  group: velero.io
+  names:
+    kind: RotationScheme
+    listKind: RotationSchemeList
+    plural: rotationschemes
+    shortNames:
+    - rs
+    singular: rotationscheme
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.keepDaily
+      name: Keep Daily
+      type: integer
+    - jsonPath: .spec.keepWeekly
+      name: Keep Weekly
+      type: integer
+    - jsonPath: .spec.keepMonthly
+      name: Keep Monthly
+      type: integer
+    - jsonPath: .spec.keepYearly
+      name: Keep Yearly
+      type: integer
+    - jsonPath: .status.rotationSchemePhase
+      name: Phase
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    - jsonPath: .status.lastValidatedTimestamp
+      name: Last Validated
+      type: date
+    name: v1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: RotationSchemeSpec defines the specification for a Velero
+              rotation scheme which can be applied to one or more schedules
+            properties:
+              keepDaily:
+                description: Number of daily backups to keep. Leave blank to keep
+                  all daily backups, or set to 1 or more to keep only the most recent
+                  N daily backups.
```

Comment:
The descriptions could be improved, given how GFS with FIFO works: https://en.wikipedia.org/wiki/Backup_rotation_scheme#Grandfather-father-son